ecommerce-ui/src/RentalItem.test.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import RentalItem from './RentalItem';
+import { rental1 } from './MockAirbnbs';
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
+  ReactDOM.render(<RentalItem
+    idx={1}
+    key={1}
+    rental={rental1}
+
+  />, div);
+  ReactDOM.unmountComponentAtNode(div);
+});
```

Comment:
Nice work on tests, but I saw no examples of snapshots. remember to create a snapshot test do something along the lines of :  
const wrapper = shallow(<RentalItem idx={1} key={1} rental={rental1}  />);
  expect(wrapper).toMatchSnapshot();